### PR TITLE
added manish tyagi in member profile

### LINKED ISSUE #3912: [Community] Member Profile: Manish Tyagi
**Current Behavior**

@money8203 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @money8203 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @money8203
- Twitter: 
- LinkedIn: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by hirentimbadiya**
**Description**

This PR fixes #3912 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 018cd8c17c003f489fd4517c8605e30480b5c6ab at: https://64088cfd29a7fb42ed407c90--layer5.netlify.app

**Comment by hirentimbadiya** [comment]
manish
@money8203

File: src/collections/members/manish-tyagi/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Manish Tyagi
+position: contributor
```
Comment (Nikhil-Ladha) [inline src/collections/members/manish-tyagi/index.mdx:3]:
```suggestion
position: Contributor
```

**Comment by hirentimbadiya** [comment]
I added your suggested changes @Nikhil-Ladha

File: src/collections/members/manish-tyagi/index.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: Manish Tyagi
+position: Contributor
+image_path: ./manish-tyagi.png
+github: money8203
+twitter: manishtyagi__
+linkedin: manish-tyagi-8b7b22217
```
Comment (Chadha93) [inline src/collections/members/manish-tyagi/index.mdx:7]:
```suggestion
linkedin: manish-tyagi-8b7b22217
badges:
 - community
```

Comment (hirentimbadiya) [reply src/collections/members/manish-tyagi/index.mdx:7]: I have added community badge to the @money8203 's profile , can you review this PR now @Chadha93

**Comment by l5io** [comment]
🚀 Preview for commit 0927c2c8444e60afbcab27bd67f721b75c5588b3 at: https://640aa86fc2513b380560bf4e--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
LGTM

**Pull request merged**